### PR TITLE
Feature/tag versions

### PR DESCRIPTION
**Pull request by olafveerman**
Ready for a review.

1. after merge into `develop`, we should test whether image gets pushed to Docker Hub with `latest-dev` tag
2. after merge into `master`, we should test whether image gets pushed with version tag

### CODE REVIEW

File: app/config/local.js.bu [hunk 1]
```diff
@@ -0,0 +1,41 @@
+module.exports = {
```
Comment (danielfdsilva) [inline app/config/local.js.bu]: There's no need to have this file

File: package.json [hunk 2]
```diff
@@ -1,25 +1,26 @@
 {
-  "name": "rural-road-accessibility--server",
+  "name": "ram-backend",
   "version": "0.4.0",
```
Comment (danielfdsilva) [inline package.json]: Version bump?

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
       - "5432:5432"
     environment:
-      POSTGRES_PASSWORD: rra
-      POSTGRES_USER: rra
-      POSTGRES_DB: rra
+      POSTGRES_PASSWORD: ram
+      POSTGRES_USER: ram
+      POSTGRES_DB: ram
     volumes:
-      - ./setup/create-test-db.sh:/docker-entrypoint-initdb.d/create-test-db.sh
```
Comment (danielfdsilva) [inline docker-compose-dev.yml]: Isn't the test db needed for when we run tests locally?

File: app/config/local.js.bu (hunk 1 above)
Comment (olafveerman) [reply app/config/local.js.bu]: Dealt with.

File: docker-compose-dev.yml (hunk 3 above)
Comment (olafveerman) [reply docker-compose-dev.yml]: Added this back.

**Comment by olafveerman** [comment]
Should be all set @danielfdsilva

**Comment by danielfdsilva** [comment]
@olafveerman all looks good. There's still the matter of the name of the datapipeline tools

**Comment by olafveerman** [comment]
The issues with the image names of `ram-datapipeline` was addressed here: https://github.com/WorldBank-Transport/ram-datapipeline/pull/30

**Pull request merged**